src/test/java/samples/android/web/AndroidWebTest.java

```diff
@@ -2,21 +2,19 @@
 
 import aquality.appium.mobile.application.AqualityServices;
 import aquality.appium.mobile.application.MobileModule;
-import org.testng.annotations.AfterGroups;
-import org.testng.annotations.BeforeGroups;
-import org.testng.annotations.Listeners;
+import org.testng.annotations.*;
 import testreport.ScreenshotListener;
 
 @Listeners(ScreenshotListener.class)
 public abstract class AndroidWebTest {
 
-    @BeforeGroups("web")
+    @BeforeClass
```

Comment:
With this change, tests would take longer, because browser will be started and exited many times. What is the purpose of this change?

Reply:
Groups don't work here, so I changed to  beforeclass

Reply:
sure they worked, 
https://dev.azure.com/aquality-automation/aquality-automation/_build/results?buildId=838&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=fbe05904-ea97-54f8-0033-906e96004c63
tests were completed faster recently...